plugin/lighthouse/handler.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/miekg/dns"
 )
 
+const LightHouse = "lighthouse"
```

Comment:
Constant names should say what the value does, or is used for, not what it is:

```suggestion
const PluginName = "lighthouse"
```

Reply:
@Jaanki  ^ Please check Stephen comment. You resolved without addressing.

Reply:
I did change it @mangelajo 
https://github.com/submariner-io/lighthouse/pull/466/commits/ec0c600fa3cb5edeb0006285eae378daf7450aa4#diff-0927cc92f1f2487ba548827cdefc8ad44f1e0308b451fe4382ec267d61425d63R29